autoload/ctrlspace/help.vim

```diff
@@ -312,8 +312,20 @@ function! s:keyHelp(key, description) abort
     endif
 endfunction
 
+function! s:sortKeyHelp(key1, key2) dict
+    if !has_key(s:descriptions, self[a:key1])
+        return 1
+    elseif !has_key(s:descriptions, self[a:key2])
+        return -1
+    endif
```

Comment:
Okay good. Though now, you should also check the case where both keys do not have descriptions, in which case `return 0` seems the most sensible. Right now in that situation, `key1` always gets shuffled to be after `key2` even if it was before it originally.

Reply:
Ah right.
I now found a much better way. No description just falls back to the empty string and is sorted naturally.